basic/src/test/java/org/jboss/hal/testsuite/test/runtime/batch/BatchJobOperations.java

```diff
@@ -0,0 +1,89 @@
+package org.jboss.hal.testsuite.test.runtime.batch;
+
+import static java.util.stream.Collectors.toSet;
+import static org.jboss.hal.testsuite.page.runtime.BatchRuntimePage.MONITORED_SERVER;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.Property;
+import org.jboss.hal.testsuite.util.ConfigUtils;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.ReadResourceOption;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+/**
+ * Encapsulates functionality of reading and handling batch job executions in management model.
+ *
+ * @author pjelinek
+ */
+public class BatchJobOperations {
+
+    private static final String
+        BATCH_JBERET = "batch-jberet",
+        HOST_NAME = ConfigUtils.getDefaultHost();
+    private final OnlineManagementClient client;
+    private final Operations ops;
+
+    public BatchJobOperations(final OnlineManagementClient client) {
+        this.client = client;
+        this.ops = new Operations(client);
+    }
+
+    public long startJob(final String deploymentName, final String jobXmlName) throws IOException {
+        ModelNodeResult result = ops.invoke("start-job", getBatchSubsystemInDeploymentAddress(deploymentName),
+                Values.of("job-xml-name", jobXmlName));
+        result.assertDefinedValue();
```

Comment:
you are right about the `assertDefinedValue()` and `assertSuccess()`, I forgot it is incorporated.